src/constants/societyNames.js

```diff
@@ -0,0 +1,22 @@
+// society Names
+const ISTELLE = 1;
```

Comment:
Might be a nit pick but why not have a single object that holds this instead of declaring 4 separate constants?

Reply:
Hmm I dunno if this way gives us a massive performance boost? @kimani-ndegwa. I will have to check back after I do a set timeout function to test 😸

Reply:
Scratch this i think I have found something we can both be happy with https://medium.com/chrisburgin/rewriting-javascript-replacing-the-switch-statement-cfff707cf045